src/bundle/Resources/views/content/tabs/versions/table.html.twig

```diff
@@ -0,0 +1,32 @@
+<table class="ez-table-data">
+    <thead>
+        <tr>
+            <th>{{ 'locationview.versions.version'|trans({}, 'locationview') }}</th>
```

Comment:
You could also set the translation domain globally for the template. It would avoid having to repeat it in every `|trans()` call, thus skipping the empty hash as the first argument.